README.md

```diff
@@ -2,18 +2,22 @@
 The Google Kubernetes Engine (GKE) Plugin allows you to deploy build artifacts to Kubernetes clusters running in GKE with Jenkins.
 
 ## Documentation
-Please see the [Google Kubernetes Engine Plugin](https://wiki.jenkins.io/display/JENKINS/Google+Kubernetes+Engine+Plugin) wiki for complete documentation.
+Please see the [Google Kubernetes Engine Plugin](docs/Home.md) wiki for complete documentation.
 
 ## Installation
-1. Download the plugin from [the Jenkins Graphite plugin bucket](https://storage.googleapis.com/gcp-jenkins/plugins/google-kubernetes-engine-latest.hpi).
-
 1. Go to **Manage Jenkins** then **Manage Plugins**.
-1. In the Plugin Manager, click the **Advanced** tab and then **Choose File** under the **Upload Plugin** section.
-1. Choose the Jenkins plugin file downloaded in Step 1.
-1. Click the **Upload** button.
+1. (Optional) Make sure the plugin manager has updated data by clicking the **Check now** button.
+1. In the Plugin Manager, click the **Available** tab and look for the "Google Kubernetes Engine Plugin".
+1. Check the box under the **Install** column and click the **Install without restart** button.
+1. If the plugin does not appear under **Available**, make sure it appears under **Installed** and is enabled.
+ 
+## Legacy Installation
+The [the Jenkins Graphite plugin bucket](https://storage.googleapis.com/gcp-jenkins/plugins/google-kubernetes-engine-latest.hpi) is no longer the source for .hpi files.
```

Comment:
This project being so new, I wonder if it's worth mentioning the plugin bucket at all anymore. Perhaps we should just mention snapshot installation.

Reply:
Fair point, I think I'll remove it then.